WORKSPACE

```diff
@@ -43,3 +43,21 @@ git_repository(
 )
 load("@io_bazel_stardoc//:setup.bzl", "stardoc_repositories")
 stardoc_repositories()
+
+
+load("@graknlabs_bazel_distribution//maven:deps.bzl", assemble_maven_deps = "deps", "maven_artifacts_with_versions")
+assemble_maven_deps()
+
+load("@rules_jvm_external//:defs.bzl", rje_maven_install = "maven_install")
```

Comment:
installing maven dependencies should be done near the code that installs pip dependencies above for consistency (`pip_deps`).

also, just like we have the macro `pip_deps`, let's have a macro `maven_deps`.

the macro can be placed in `//maven:deps.bzl` just like how we did it for pip:`//pip:deps.bzl`

Reply:
@lolski addressed in https://github.com/graknlabs/bazel-distribution/pull/293/commits/110644cefa7e51c2cb240377191b20a9444c0fa0